### PR TITLE
Deprecate old examples and refer to newer versions

### PR DESCRIPTION
**Pull request by marcosbc**

### CODE REVIEW

File: dotnet-core-with-mysql/todo/README.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
-# .NET Core Application with DB (MySQL): Simple asp net core todo app
+# .NET Core Application with DB (MySQL): Simple asp net core todo app (deprecated)
+
+## Deprecation notice
```
Comment (vikram-bitnami) [inline dotnet-core-with-mysql/todo/README.md]: Maybe this should be in bold and in a box of some kind so it stands out.

File: README.md [hunk 2]
```diff
@@ -6,11 +6,18 @@ It contains the following examples:
...
-* [A MySQL-based application (Ghost)](generic-with-mysql/ghost)
 * [A Java/Tomcat/PostgreSQL application (JasperReports)](java-tomcat/jasperreports)
-* [A Node.js/MongoDB application](nodejs-with-nosql/todo/)
 * [A PHP/VueJS application](php/todo/)
 
+The following examples can be found in external repositories:
+
+* [WordPress, a PHP/MySQL application](https://github.com/bitnami-labs/stacksmith-wordpress)
+* [Drupal, a PHP/MySQL application](https://github.com/bitnami-labs/stacksmith-drupal)
+* [Jenkins, a Java/Tomcat application without database](https://github.com/bitnami-labs/stacksmith-jenkins)
```
Comment (vikram-bitnami) [inline README.md]: without a database

File: README.md [hunk 3]
```diff
@@ -6,11 +6,18 @@ It contains the following examples:
...
+* [Ghost, a Node.js/MySQL application](https://github.com/bitnami-labs/stacksmith-ghost) (previously located at [generic-with-mysql/ghost](generic-with-mysql/ghost/))
+* [A Node.js/MongoDB To-Do example application](https://github.com/bitnami-labs/stacksmith-nodejs-with-nosql) (previously located at [nodejs-with-nosql/todo](nodejs-with-nosql/todo/))
```
Comment (vikram-bitnami) [inline README.md]: [An example Node.js/MongoDB application]

File: README.md [hunk 4]
```diff
@@ -6,11 +6,18 @@ It contains the following examples:
...
+* [A .NET Core/MySQL To-Do example application](https://github.com/bitnami-labs/stacksmith-dotnet-with-mysql) (previously located at [dotnet-core-with-mysql/todo](dotnet-core-with-mysql/todo/))
+
```
Comment (vikram-bitnami) [inline README.md]: [An example .NET Core/MySQL application]

File: dotnet-core-with-mysql/todo/README.md [hunk 5]
```diff
@@ -1,4 +1,10 @@
-# .NET Core Application with DB (MySQL): Simple asp net core todo app
+# .NET Core Application with DB (MySQL): Simple asp net core todo app (deprecated)
```
Comment (vikram-bitnami) [inline dotnet-core-with-mysql/todo/README.md]: ASP.NET Core to-do application

File: README.md (hunk 3 above)
Comment (beltran-rubo) [reply README.md]: I would not add the previous location, it could be confusing

File: dotnet-core-with-mysql/todo/README.md (hunk 1 above)
Comment (marcosbc) [reply dotnet-core-with-mysql/todo/README.md]: GitHub does not offer this functionality, apart from being able to create a table with only one column and one row. Therefore I think we should leave it like this, as it is clear enough to users that it is deprecated.

File: README.md (hunk 3 above)
Comment (marcosbc) [reply README.md]: Ok! I will implement both suggestions.

File: README.md (hunk 4 above)
Comment (marcosbc) [reply README.md]: Done

File: dotnet-core-with-mysql/todo/README.md (hunk 5 above)
Comment (marcosbc) [reply dotnet-core-with-mysql/todo/README.md]: Done

**Pull request merged**